neural_compressor/tensorflow/quantization/config.py

```diff
@@ -103,6 +103,13 @@ def register_supported_configs(cls) -> List[OperatorConfig]:
         supported_configs.append(OperatorConfig(config=static_quant_config, operators=operators))
         cls.supported_configs = supported_configs
 
+    @classmethod
+    def get_config_set_for_tuning(
+        cls,
+    ) -> Union[None, "StaticQuantConfig", List["StaticQuantConfig"]]:  # pragma: no cover
```

Comment:
@zehao-intel , we need to update it after the TF enables the auto-tune.

Reply:
Get it, thanks.